core/src/test/scala/unit/kafka/server/ClientQuotasRequestTest.scala

```diff
@@ -165,7 +166,9 @@ class ClientQuotasRequestTest(cluster: ClusterInstance) {
     ))
   }
 
-  @ClusterTest(types = Array(Type.ZK)) // No SCRAM for Raft yet
+  // TODO: KAFKA-17630 -  Convert ClientQuotasRequestTest#testClientQuotasForScramUsers to kraft
+  @Disabled
```

Comment:
The `@Disabled` annotation takes a String value. Can you put the JIRA ticket in there?